src/apps/vpn/appfeaturelistcallback.h

```diff
@@ -129,6 +129,14 @@ bool FeatureCallback_splitTunnel() {
     if (VersionUtils::compareVersions(shellVersion, "3.34") < 0) {
       return false;
     }
+  } else if (desktop.contains("KDE")) {
+    QString kdeVersion = LinuxDependencies::kdeFrameworkVersion();
+    if (kdeVersion.isNull()) {
+      return false;
+    }
+    if (VersionApi::compareVersions(kdeVersion, "5.75") < 0) {
```

Comment:
Can you write a comment about this 5.75?

Reply:
Hope the comment is clear enough. Otherwise we can probably link to https://github.com/KDE/kio/commit/ea621368bbcb0b5fff804a6e39ff901c69028140 directly?